gson/src/test/java/com/google/gson/internal/bind/DefaultDateTypeAdapterTest.java

```diff
@@ -223,6 +225,27 @@ public void testUnexpectedToken() throws Exception {
     }
   }
 
+  @Test
+  public void testGsonDateFormat() {
+    // Set the default timezone to UTC
+    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
Please restore the timezone in a `finally` block as well to not affect the other tests in any way, see for example how the `assertFormattingAlwaysEmitsUsLocale` method here does this.